site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,233 @@
+
+Citra has been able to run many games at playable speeds for a while now. However, it was always lacking
+something which the 3DS has: the ability to play with friends. In order to it make a reality, our developers [jroweboy](https://github.com/jroweboy), [B3n30](https://github.com/B3N30), [Subv](https://github.com/subv),
+and [JayFoxRox](https://github.com/jayfoxrox) worked tirelessly for months to reverse engineer workings
+of the 3DS local wireless system. Thanks to their efforts, Citra now emulates _local wireless multiplayer_
+over the internet. This allows you to play your favorite games with your virtual friends across the world.
+Later [jroweboy](https://github.com/jroweboy), [B3n30](https://github.com/B3n30), and few others
+worked on various feature improvements and bug fixes.
+
+### Features
+Currently, the multiplayer features include:
+
+* We now have multiple servers, maintained by our very own [FlameSage](https://community.citra-emu.org/u/flamesage/summary)
+and many other community members at various locations around the globe, so that you can meet others
+online.
+* You can now create rooms which can hold up to 16 players and even secure them with a password.
+* You can now set a preferred game for your room.
+* Since we now have many number of servers, you can also filter the servers to suit your needs.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/server-list.png"
+    title="Sooo many servers to choose from!" >}}
+
+Since its initial release in Canary, developers have listened to the community and tried very hard
+to fix bugs affecting many games which made them incompatible. They have also added other features
+to make the multiplayer experience richer.
+
+### Compatible Titles
+We are proud to announce that multiplayer currently works in the following list of titles:
+
+* Dragon Quest Monsters: Joker 3 Professional
+* Luigi's Mansion: Dark Moon
+* Mario Party: The Top 100
+* Monster Hunter 3G & 3 Ultimate
+* Monster Hunter 4, 4G & 4 Ultimate
+* Monster Hunter X
+* Monster Hunter XX
+* New Super Mario Bros. 2
+* Pokémon Omega Ruby / Alpha Sapphire
+* Pokémon Sun / Moon
+* Pokémon Ultra Sun / Ultra Moon
+* Pokémon Virtual Console
+* Super Smash Bros.
+* The Legend of Zelda: Tri Force Heroes
+
+### Incompatible Titles
+For various reasons, the following games were tested and do not work:
+
+* Asphalt Assault 3D
+* Code of Princess (*a fix is in works*)
+* Dragon Quest Monsters: Terry's Wonderland 3D
+* F1 2011
+* Kirby Fighters Deluxe
+* Kirby Triple Deluxe
+* Mario Party Island Tour
+* Mario Party Star Rush
+* Mario Kart 7
+* Planet Crashers
+* Resident Evil: The Mercenaries 3D
+* Ridge Racer 3D
+* Sonic Generations
+* Street Fighter IV
+* Tetris Ultimate
+
+***Also note that Download Play and Spotpass titles do not work as they are beyond the scope of current
+local multiplayer***.
+
+**Note**: Although multiplayer is supported, some games might still experience issues with some games.
```

Comment:
I don't think the "**Note**" here is necessary, given there's already a note right above this.